tests/openvino/test_install.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import pytest
+
+from tests.common.helpers import run_install_checks
+
+
+@pytest.fixture(name="venv_type",
+                params=["virtualenv", "venv"])
+def venv_type_(request):
+    return request.param
+
+
+@pytest.fixture(name="package_type",
+                params=["install", "develop", "sdist", "bdist_wheel",
+                        "pip_pypi", "pip_local", "pip_e_local", "pip_git_develop"])
+def package_type_(request):
+    return request.param
+
+
+def test_install(tmp_venv_with_nncf, tmp_path, package_type):
+    run_install_checks(tmp_venv_with_nncf, tmp_path, package_type, test_dir='openvino')
```

Comment:
please check this function since it has some PT and TF-specific logic that should be extended as well.

Reply:
Thanks! Done!